os-config/ubuntu-credentials.robot

```diff
@@ -3,7 +3,7 @@
 # across installations. At least hostname has been defined, and
 # username and hostname variables are used in the prompts. Before,
 # we needed to change them in prompts as well.
-${UBUNTU_USERNAME}=         ubuntu
+${UBUNTU_USERNAME}=         user
```

Comment:
I think that the global standard is that the username is ubuntu. If on your platform it is different it should be changed on DUT to `ubuntu` and not in the code as it most propably will brake tests for other platforms.

Reply:
git add . incident
all good now